src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const collection = document.querySelectorAll('.population');
+
+const values = [...collection]
+  .map(item => +(item.innerText.replaceAll(',', '')));
+
+const total = values.reduce((sum, x) => sum + x);
+const average = total / values.length;
+
+document.querySelector('.total-population').innerText
+  = total.toLocaleString();
+
+document.querySelector('.average-population').innerText
+  = average.toLocaleString();
```

Comment:
it looks much better if you move document.querySelector('.total-population') and document.querySelector('.average-population') to own variables

Reply:
understood, thanks